syndesis/src/modules/data/shared/VirtualizationNavBar.tsx

```diff
@@ -12,37 +32,53 @@ import routes from '../routes';
  * 4. a link to the page that dispays metrics
  *
  */
-export default class VirtualizationNavBar extends React.Component {
+export default class VirtualizationNavBar extends React.Component<
+  IVirtualizationDetailRouteState,
```

Comment:
A component interface should not have `Route` in its name. That's a convention we use only for interfaces to be fed to the `WithRouteData` component, which should be used in Pages only.

In this case I think you can simply pass the virtualization component you need, and drop the `WithRouteData` at line 41

Reply:
Thanks @riccardo-forina.. great point. Updated to just pass the component and removed WithRouteData.